lib/hmmlearn/hmm.py

```diff
@@ -307,18 +309,215 @@ def _do_mstep(self, stats):
                                      (cvweight + stats['post'][:, None, None]))
 
 
-_MULTINOMIALHMM_DOC_SUFFIX = """
+
+class MultinomialHMM(BaseHMM):
+    """
+    Hidden Markov Model with multinomial emissions
+
+    Attributes
+    ----------
+    n_features : int
+        Number of possible symbols emitted by the model (in the samples).
+
+    monitor_ : ConvergenceMonitor
+        Monitor object used to check the convergence of EM.
+
+    startprob_ : array, shape (n_components, )
+        Initial state occupation distribution.
+
+    transmat_ : array, shape (n_components, n_components)
+        Matrix of transition probabilities between states.
+
+    emissionprob_ : array, shape (n_components, n_features)
+        Probability of emitting a given symbol when in each state.
+
+    Examples
+    --------
+    >>> from hmmlearn.hmm import MultinomialHMM
+    """
+
+    def __init__(self, n_components, n_trials, n_features=2,
```

Comment:
Should n_features be an init arg?  (Does defaulting to 2 really help?)  Should it not be inferred from e.g. the shape of emissionprob (if set) or the fitted time series, basically entirely form _check_and_set_n_features?

Reply:
I see what you're saying now. For some reason I thought the other HMMs also had n_features as an init param (maybe because of the comment in this [line](https://github.com/hmmlearn/hmmlearn/blob/1c732510f2e2afa40bc6e1ed3a387cbccff8bbdd/lib/hmmlearn/hmm.py#L43)), but I'm looking at them and that's not the case. So yeah, I'll remove this param.